sdk/communication/azure-communication-rooms/azure/communication/rooms/_rooms_client.py

```diff
@@ -152,88 +144,35 @@ def delete_room(
         """
         return self._rooms_service_client.rooms.delete(room_id=room_id, **kwargs)
 
-    @overload
-    def update_room(
-        self,
-        communication_room: CommunicationRoom,
-        **kwargs: Any
-    ) -> CommunicationRoom:
-        """Update a valid room's attributes by passing in the updated CommunicationRoom.
-
-        :param communication_room: An instance of CommunicationRoom. This is a positional-only parameter.
-          Please provide either this or individual keyword parameters.
-        :type communication_room: CommunicationRoom
-        :rtype: ~azure.communication.rooms.CommunicationRoom
-        :raises: ~azure.core.exceptions.HttpResponseError, ValueError
-        """
-
-    @overload
+    @distributed_trace
     def update_room(
         self,
         *,
         room_id: str,
-        valid_from: Optional[datetime] = None,
-        valid_until: Optional[datetime] = None,
+        valid_from: Optional[datetime],
```

Comment:
is `valid_from` a required input? I think it should still default to `None` to be consistent with how we treat optional params

Reply:
we've decided for valid_from to be a required input for the time being since the service will throw exception when one of valid_from or valid_until is not passed in.

Reply:
Yep that's my fault - I forgot to remove the `Optional` type hint when I removed the default None.